src/sidebar/panel.js

```diff
@@ -69,11 +69,16 @@ ClassicEditor.create(document.querySelector('#editor'), {
         }
         ignoreNextLoadEvent = false;
       });
-
+      
+      savingIndicator.onclick = () => {
+        enableSyncAction(editor);
```

Comment:
I believe this should be deactivated during saving (as for enableSync)

Reply:
Nevermind it seems to be handled in the enableSyncAction itself: https://github.com/utkarsh858/notes/blob/6135f7841191671facbfcefd12f91d5f033c9e46/src/sidebar/panel.js#L238